sentinel-core/log/metric/metric_node.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <stdint.h>
+#include <string>
+
+namespace Sentinel {
+namespace Metric {
+
+class MetricNode {
```

Comment:
@zyfjeff I've implemented the `MetricNode` class as `MetricItem` in master branch. After merging this PR we can replace the legacy `MetricNode` with `MetricItem` class.

Reply:
Is it better to merge the master directly in this PR and modify it?

Reply:
Yes, it's more recommended to modify this in this PR.